lib/base/novacom.js

```diff
@@ -323,6 +322,7 @@ const fs = require('fs'),
             if (devices.length < 1) {
                 setImmediate(next, errHndl.getErrMsg("UNMATCHED_DEVICE", key, value));
             } else {
+                console.log("Set target device to " +  value);
```

Comment:
move position "Connecting target"

Reply:
Removed this print. I'll add the device target information in CLI status message format work.